packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -281,9 +291,8 @@ const Link = React.forwardRef(({ element, children, ...attributes }, forwardedRe
 
   const handleSave = (e) => {
     e.stopPropagation();
-    const data = new FormData(e.target);
 
-    if (data.get('url') === '') {
+    if (formData.url === '') {
```

Comment:
Maybe this conditional is not longer necessary if you are disabling the button, right?

Reply:
true! 👍